frontend/src/views/LevelView.vue

```diff
@@ -64,20 +69,24 @@ const createLevel = async () => {
       ErrorMessage('You need to fill all the fields')
       return
     }
-    if (newLevel.value.maxCoeficient == 0) {
+    if (newLevel.value.maxCoefficient == 0 && newLevel.value.maxCoefficient < 101) {
```

Comment:
What do you mean with this condition? You should just check if the maxCoeff is fewer than zero or greater than 100, no?